utils/persistent_data_frame.py

```diff
@@ -8,19 +8,24 @@
 class PersistentDataFrame:
     """
     A logging class managing loading, updating and storing the experiments data.
-    extra feature: this class is atomic and fit for concurrent processing.
+    This class is supports atomic operations and is fit for concurrent processing.
+    However, for multiple operations, please explicitly acquire a lock.
 
     Usage:
     ------
-
-    Init:
-    >>> persistent_df = PersistentDataFrame(csv_path)
+    Init (this csv may be accessed by multiple processes):
+    >>> per_df = PersistentDataFrame(csv_path)
```

Comment:
is the rename really necessary?

Reply:
The other examples I moved used this name so I renamed the others to match it